scripts/README.md

```diff
@@ -16,9 +16,11 @@ Linux (tested on Ubuntu 18.04 and 20.04):
     - `build_plugin_pkg()`: build a tarball containing the plugin
 
 - `build_plugin_linux.sh`: build the plugin on a linux computer (relies on apt-get to install required dependencies)
-    - arguments: Trelis/Cubit version to build the plugin for (only supported yet `17.1.0`)
+    - arguments: 
+        - 1 Trelis/Cubit version to build the plugin for (only supported yet `17.1.0`and Cubit `2020.2`)
+        - 2 path to Trelis/Cubit package folder (containing Trelis install package and SDK)
 - `build_plugin_docker_ubuntu.sh`: build the plugin in a docker container 
     - arguments: Arguments have to be provided in order
         - 1 docker image (tested on `ubuntu:18.04` and `ubuntu:20.04`)
         - 2 local path to the folder containing the Trelis/Cubit install deb and SDK
-        - 3 version of Cubit Trelis to build the plugin for (only tested with `17.1.0`)
+        - 3 version of Cubit Trelis to build the plugin for (only tested with `17.1.0` and Cubit `2020.2`)
```

Comment:
Can we flip args 2 and 3 here so the order is more consistent between the docker and Linux scripts?